R/pkg/R/functions.R

```diff
@@ -488,19 +488,35 @@ setMethod("initcap",
             column(jc)
           })
 
-#' isNaN
+#' isnan
 #'
-#' Return true iff the column is NaN.
+#' Return true if the column is NaN.
 #'
-#' @rdname isNaN
-#' @name isNaN
+#' @rdname isnan
+#' @name isnan
 #' @family normal_funcs
 #' @export
-#' @examples \dontrun{isNaN(df$c)}
-setMethod("isNaN",
+#' @examples \dontrun{isnan(df$c)}
+setMethod("isnan",
           signature(x = "Column"),
           function(x) {
-            jc <- callJStatic("org.apache.spark.sql.functions", "isNaN", x@jc)
+            jc <- callJStatic("org.apache.spark.sql.functions", "isnan", x@jc)
+            column(jc)
+          })
+
+#' isnull
+#'
+#' Return true if the column is NULL.
+#'
+#' @rdname isnull
+#' @name isnull
+#' @family normal_funcs
+#' @export
+#' @examples \dontrun{isnull(df$c)}
+setMethod("isnull",
```

Comment:
In the context of DataFrame column, "null" means missing value, which I think NA in R means. When we read a column from a DataFrame to R side, null will be converted to NA, see the code at https://github.com/apache/spark/blob/master/R/pkg/R/deserialize.R#L115